pages/pil/pdf/content/standard-conditions.js

```diff
@@ -0,0 +1,92 @@
+  possible when it appears either that the severity limit of any procedure listed in the project licence or that
+  the constraints upon adverse effects described in the project licence have been or are likely to be
+  exceeded.`,
+
+  `The licence holder shall ensure that suitable arrangements exist for the care and welfare of animals
+  during any period when the personal licence holder is not in attendance.`,
+
+  `The licence holder shall ensure that, whenever necessary, veterinary advice and treatment are
+  obtained for the animals in his or her care.`,
+
+  `The licence holder shall ensure that all cages, pens or other enclosures are clearly labelled. The
+  labelling must be such as to enable Inspectors, named veterinary surgeons and named animal care and
+  welfare officers to identify the number of the project licence authorising the procedures, the project
+  licence protocol in which the animals are being used, the date the protocol was started, and the
+  responsible personal licence holder.`,
+
+  `In order to ensure that regulated procedures are performed competently, the licence holder shall
+  not apply regulated procedures unless given the appropriate level of supervision by the project licence
+  holder or an experienced personal licence holder deputed by him/her for such time as may be needed to
+  achieve competence.`,
+
+  `The licence holder is authorised to delegate to assistants, who do not themselves possess the
+  requisite personal licence authority but are under his or her control, the delegable tasks which form an
+  integral part of the regulated procedures the licence holder is authorised to perform by this licence. The
+  tasks must not require technical knowledge or skill, and delegation shall be in accordance with any
+  relevant guidance published by the Secretary of State under section 21.`,
+
+  `The licence holder must take all reasonable steps to ensure appropriate personal and project
+  licence authorities exist before performing regulated procedures. The licence holder must be aware of
+  the nature of the authorities given by this licence and the project licence, and of the conditions of issue
+  attached to the licences.`,
+
+  `The licence holder shall maintain a record of all animals on which procedures have been carried
+  out, including details of supervision and declarations of competence by the project licence holder as
+  appropriate. This record shall be retained for at least five years and shall, on request, be submitted to the
+  Secretary of State or made available to an Inspector.`,
+
+  `The licence holder must give any necessary assistance to inspectors carrying out visits by virtue of
+  section 18(2A)(b) ; and to experts of the European Commission carrying out duties under Article 35 of
+  the Animals Directive.`,
+
+  `The licence remains the property of the Secretary of State, and shall be surrendered to him on
+  request.`
+];
```

Comment:
What's the logic behind this being an array and not just a big block of markdown?

Reply:
I was copying how @joefitter did his conditions here: https://github.com/UKHomeOffice/asl-projects/blob/master/client/constants/standard-conditions.js so that I could re-use the purple inset styling.

Reply:
👍